README.md

```diff
@@ -124,58 +136,137 @@ export type PluginOptions = {
 
 ### rules
 
-An array of rules consisting of these properties:
+The rules that define whether a visit will be considered a fragment visit. Each rule consists of
+mandatory `from` and `to` URL patterns, an array `fragments` of selectors, as well as an optional
+`name` of this rule to allow precise styling. The from/to patterns are converted to regular
+expressions by [path-to-regexp](https://www.npmjs.com/package/path-to-regexp).
+
+```js
+{
+  rules: [
+    {
+      from: '/users/:filter?',
+      to: '/users/:filter?',
+      fragments: ['#users'],
```

Comment:
Even though regular expressions are supported, I would not advertise them. Using strings or arrays of strings and let path-to-regex do the heavy lifting will prevent many problems.

Reply:
Updated to: 'The from/to patterns are interpreted and follow the syntax established by path-to-regexp'. Does that make sense?